tests/integration/sync/content/syncContent.test.ts

```diff
@@ -0,0 +1,127 @@
+  const data1 = await loadAllEnvData(environmentId1, { include: ["types"] })
+    .then(async res => ({ ...(await loadVariantsByItemCodename(environmentId1, itemCodenames)), types: res.types }))
+    .then(prepareReferences);
+  const data2 = await loadAllEnvData(environmentId2, { include: ["types"] })
+    .then(async res => ({ ...(await loadVariantsByItemCodename(environmentId2, itemCodenames)), types: res.types }))
+    .then(prepareReferences);
+
+  expectSameAllEnvData(data1, data2, { include: ["variants"] });
+};
+
+describe("Sync two environments with credentials", () => {
+  it.concurrent(
+    "sync-content source environment to target environment without linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --items sync_item_1 --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, ["sync_item_1"]);
+    }),
+  );
+
+  it.concurrent(
+    "Sync source environment to target environment directly from source environment with linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --items sync_item_1 --depth 2 --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, [
+        "sync_item_1",
+        "sync_linked_item_1",
+      ]);
+    }),
+  );
+
+  it.concurrent(
+    "Sync source environment to target environment directly from source environment get item byContentType with linked item",
+    withTestEnvironment(SYNC_TARGET_TEST_ENVIRONMENT_ID, async (environmentId) => {
+      const command =
+        `sync-content run -s=${SYNC_SOURCE_TEST_ENVIRONMENT_ID} --sk=${API_KEY} -t=${environmentId} --tk=${API_KEY} -l=default --byTypesCodenames no_change_linked_type no_change_base_type --verbose --skipConfirmation`;
+
+      await runCommand(command);
+
+      await expectSameSyncContentEnvironments(environmentId, SYNC_SOURCE_TEST_ENVIRONMENT_ID, [
+        "sync_item_1",
+        "sync_draft_linked_item_1",
+        "sync_item_2",
+        "sync_linked_item_2",
+        "sync_draft_linked_item_2",
+        "sync_draft_item_1",
+        "sync_linked_item_1",
+      ]);
+    }),
+  );
+});
+
+describe.only("Sync environment from folder", () => {
+  const relativeFolderPath = "./tests/integration/sync/content/data";
+  const relativeContentZipPath = `${relativeFolderPath}/sourceContent.zip`;
+
+  beforeAll(async () => {
+    await fsPromises.mkdir(relativeFolderPath, { recursive: true }); // recursive skips already created folders
```

Comment:
The `Sync content from folder` test doesn't create any files so there is no need to create the folder before its run. For now it is enough to create it just in the `export sync content` and if more tests will need it in the future, we will adjust later accordingly.